crates/iota-indexer/src/test_utils.rs

```diff
@@ -90,6 +95,10 @@ pub async fn start_test_indexer_impl(
         IndexerTypeConfig::Reader {
             reader_mode_rpc_url,
         } => {
+            if let Some(db_init_hook) = db_init_hook {
+                db_init_hook(&store);
+            }
```

Comment:
This is called in each match arm with the same store, just move it before the match?

Reply:
Done b573cb828cd56120bc0ec7c3fdc56d055f060b7c